src/Database/Esqueleto/Internal/Sql.hs

```diff
@@ -17,7 +17,7 @@
 module Database.Esqueleto.Internal.Sql
   ( -- * The pretty face
     SqlQuery
-  , SqlExpr
+  , SqlExpr(..)
```

Comment:
Is it necessary to export `SqlExpr`'s constructors?

Reply:
Unfortunately it was the only way I could figure out to make [unsafeSqlAggregateFunction](https://github.com/bitemyapp/esqueleto/pull/76/files#diff-c901387a600e739ea0b320c45ad0882bR67) work. 

Besides, is there a specific reason not to export it from an .Internal module? The usual recommendation is to expose as much as possible through them as it allows for cleaner implementations of custom functions like this  and .Internal modules usually come with an implied "here be dragons, user discretion is advised" anyway.

Reply:
@Philonous that's fine by me, I just wanted to make sure I understood.